PeerioTechnologies/peerio-mobile
# fix: file share toolbar overlapping files in files list [branch ch5247]

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/5247/mobile-file-share-toolbar-covers-the-bottom-most-file-in-files-list-when-sharing-in-a-chat
#### Testing instructions  
Steps to reproduce:
1- Be logged into an account
2- Have enough files and folders in Files list to enable scrolling
3- Enter into a chat, click on the "+" button to share a file and select "Share from Peerio Files"
4- Try to scroll to the bottom of the list

Before fix: The bug is that the bottom most file does not show because it is covered by the toolbar which has the buttons "cancel" and "share"

After fix: bottom most file element should show.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
